app/react/App/DropdownMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { I18NLink, t } from 'app/I18N';
+import { Icon } from 'UI';
+import React, { useRef, useState, useCallback } from 'react';
+import { useOnClickOutsideElement } from 'app/utils/useOnClickOutsideElementHook';
+
+export type DropdownMenuProps = {
+  link: any;
+};
+
+export function DropdownMenu({ link }: DropdownMenuProps) {
+  const [showing, setShowing] = useState(false);
+  const dropdownRef = useRef(null);
+  const onClickOutside = useCallback(() => {
+    setShowing(false);
+  }, []);
+
+  useOnClickOutsideElement<HTMLLIElement>(dropdownRef, onClickOutside);
+
+  return (
+    <li className="menuNav-item" key={link.get('_id')} ref={dropdownRef}>
```

Comment:
I don't think this is working as expected. I'm getting the `Warning: Each child in a list should have a unique "key" prop.` in the console.

Reply:
Fixed